src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/hooks.ts

```diff
@@ -34,7 +35,56 @@ export const useOutcomeStep = (motionData: ColonyMotion | null | undefined) => {
     ];
   }, [motionData]);
 
+  const stakingData = useMemo(() => {
+    if (!motionData) return undefined;
+
+    const { usersStakes } = motionData;
+
+    const stakers = usersStakes.flatMap((userStakes) => {
+      const { address, stakes } = userStakes;
+
+      const yay = Number(stakes.percentage.yay);
+      const nay = Number(stakes.percentage.nay);
+      const addresses: UserAvatarsItem[] = [];
+
+      if (yay > 0 && nay > 0) {
+        addresses.push({ address, voteCount: '1', vote: MotionVote.Yay });
+        addresses.push({ address, voteCount: '1', vote: MotionVote.Nay });
+      } else if (yay > 0) {
+        addresses.push({ address, voteCount: '1', vote: MotionVote.Yay });
+      } else if (nay > 0) {
+        addresses.push({ address, voteCount: '1', vote: MotionVote.Nay });
+      }
```

Comment:
This can be written in a simpler way:
```
if (yay > 0) {
  addresses.push({ address, voteCount: '1', vote: MotionVote.Yay });
}

if (nay > 0) {
  addresses.push({ address, voteCount: '1', vote: MotionVote.Nay });
}
```